db_model/orm_model.py

```diff
@@ -116,6 +116,7 @@ class BuildAction(Base):
     run_id = Column(Integer,
                     ForeignKey('runs.id', deferrable=True, initially="DEFERRED",
                                ondelete='CASCADE'))
+    # hash of the build command
```

Comment:
I can generate the schema upgrade script, but if we go with that all of the users need to change their CodeChecker versions to the latest, which I do not wanted to do in 5.6.1.
If we go with the db upgrade release version 5.7 might be better, because this is minor change in the database.